client/src/pages/TourPage/TourPage.js

```diff
@@ -265,15 +272,12 @@ const TourPage = () => {
           ) : null}
         </Container>
 
-        {!matchesMediaQuery ? (
+        {!screenWidth ? (
           <Slide in={!inView} direction="up" mountOnEnter unmountOnExit>
             <FloatingDialog>
               <Container>
                 <Stack direction="row" justifyContent="space-between" alignItems="center">
-                  <Cost>
-                    {currency.eur}
-                    {cost.eur}
-                  </Cost>
+                  <Cost>€{currentPrice}</Cost>
```

Comment:
It has to inherit value from select. As I understood you'll do it later?

Reply:
i dont get your point.

Reply:
discussed